website/public/js/services/guideServices.js

```diff
@@ -239,6 +239,7 @@ function($rootScope, User, $timeout, $state) {
     if (!updateFn) return; // only run after user has been wrapped
     watcher(); // deregister watcher
     if (window.env.IS_MOBILE) return; // Don't show tour immediately on mobile devices
+    if (User.user.items.gear.owned['weapon_warrior_0']) return;//If the user has the training sword, then don't show the intro tour
```

Comment:
I think for now, canceling the tour might be simplest, but if you can make
it skip past the reward step instead, that would be helpful.

And yes, if people start on the mobile site, we shouldn't show the tour to
them.

On Thu, Apr 9, 2015 at 5:43 AM, Blade Barringer <notifications@github.com>
wrote:

> In website/public/js/services/guideServices.js
> <https://github.com/HabitRPG/habitrpg/pull/4987#discussion_r28057853>:
>
> > @@ -239,6 +239,7 @@ function($rootScope, User, $timeout, $state) {
> >      if (!updateFn) return; // only run after user has been wrapped
> >      watcher(); // deregister watcher
> >      if (window.env.IS_MOBILE) return; // Don't show tour immediately on mobile devices
> > +    if (User.user.items.gear.owned['weapon_warrior_0']) return;//If the user has the training sword, then don't show the intro tour
>
> @TheHollidayInn <https://github.com/TheHollidayInn> I think we should
> still have the check for the training sword, since it's possible to halt
> the tour, buy the sword and then resume the tour. It of course, would not
> be joined with the mobile check.
>
> I want @lemoness <https://github.com/lemoness>'s opinion on this, but
> wouldn't it be preferable to skip/change the rewards part of the tour if a
> sword has already been purchased, rather than canceling the whole tour?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/4987/files#r28057853>.
>